modules/ROOT/pages/tutorial/tutorial-sso-configuration.adoc

```diff
@@ -341,14 +365,100 @@ dbms.security.oidc.google.token_params=client_secret=GOCSPX-v4cGkygPJvm3Sjjbc0hv
+
+.. Change the configuration to allow `native` authentication (for use only by the `admin` user):
+
+[source, properties]
+----
+dbms.security.authentication_providers=oidc-google, native
+----
+
+.. Set the `dbms.security.require_local_user` configuration setting to `true` in the _neo4j.conf_ file. This will switch to User Auth Providers mode whereby users can only authenticate and authorize if they have a corresponding Auth Provider in the database.
+
+[source, properties]
+----
+dbms.security.require_local_user=true
+----
+
+
+.. Create a user who can authenticate and authorize using `oidc-google` (they will not be able to authenticate using `native` (username and password) authentication, only `oidc-google`):
+
+[source, cypher, role=noplay]
+----
+CREATE USER jake
+SET HOME DATABASE 'jakesHomeDb'
+SET AUTH 'oidc-google' {SET ID 'jakesUniqueGoogleUserId'} // this must match the claim that you configured via dbms.security.oidc.google.claims.username
+----
+
+.. Grant the user roles using native authorization.
+
+[source, cypher, role=noplay]
+----
+GRANT ROLE reader TO jake
+----
+
+The user will implicitly receive `native` authorization because `native` is in the list of authorization providers and you have explicitly granted the user a `ROLE`.
+
+You can now disable native authentication for the database completely:
+
+[source, properties]
+----
+dbms.security.authentication_providers=oidc-google
+----
+
+
+
+. Alternatively, if you don't use Auth Providers, you can temporarily enable `native` authentication to create an SSO-authenticated admin user `alice` who can then create other users who can only authenticate using SSO.
+
+To do this we temporarily enable native authentication:
+
+[source, properties]
+----
+dbms.security.authentication_providers=oidc-google, native
+----
+
+Then, create an SSO-authenticated `admin` user (in this example an equivalent of `alice@neo4j-test.com` must be set up in the Google SSO provider and their credentials must be known):
+
+[source]
+----
+CREATE USER `alice@neo4j-test.com` SET PASSWORD 'secretpassword';
+GRANT ROLE admin to `alice@neo4j-test.com`;
+----
+
+Disable native authentication for the database to prevent users logging in with username and password:
+
+[source, properties]
+----
+dbms.security.authentication_providers=oidc-google
+----
+
+. Log via Google SSO as `alice@neo4j-test.com`, the `admin` user
+
+. Use this admin user to create other users who can only authenticate using `oidc-google` and will receive the roles granted to them using `native` authorization.
+
+
+[source, cypher, role=noplay]
+----
+CREATE USER jakesUniqueGoogleUserId // this must match the claim that you configured via dbms.security.oidc.google.claims.username
```

Comment:
in this case `this` might actually make sense as there is only one thing being set :P

Reply:
might still be better to change it though 🤷